tools/helper.psm1

```diff
@@ -0,0 +1,319 @@
+
+$MinimalSDKVersion = '2.1.300'
+$IsWindowsEnv = [System.Environment]::OSVersion.Platform -eq "Win32NT"
+$RepoRoot = (Resolve-Path "$PSScriptRoot/..").Path
+$LocalDotnetDirPath = if ($IsWindowsEnv) { "$env:LocalAppData\Microsoft\dotnet" } else { "$env:HOME/.dotnet" }
+
+<#
+Synopsis: Get the path of the currently running powershell executable.
```

Comment:
If this is meant to be comment help, it should be `.SYNOPSIS` with the help text on the next line. This applies to the other changes in this file.

I believe `Invoke-Build` uses this syntax for build tasks as it has it's own system, but maybe shouldn't be used here.

Reply:
Ah, I see. I was simply copying the pattern used in `PSReadLine.build.ps1`. I thought it's just a quick way to show a 'what-it-is' comment. I will update those.

Reply:
Updated to use the help comment.
```
<#
.SYNOPSIS
    ...
#>
```